### ISSUE TITLE
Unable to play a just started live broadcast

### ISSUE THREAD

**Issue by crazytoad** [google/shaka-player#1150]
**Have you read the [FAQ](https://goo.gl/JE1Sy5) and checked for duplicate issues**:
yes

**What version of Shaka Player are you using**:
2.2.6

**Can you reproduce the issue with our latest release version**:
yes

**Can you reproduce the issue with the latest code from `master`**:
yes

**Are you using the demo app or your own custom app**:
custom app

**If custom app, can you reproduce the issue using our demo app**:
yes

**What browser and OS are you using**:
chrome 62 macOS 10.12.6

**What are the manifest and license server URIs**:
*(you can send the URIs to <shaka-player-issues@google.com> instead, but please use GitHub and the template for the rest)*
Email sent

**What did you do?**
Connected to a live stream the moment it started broadcasting and the first manifest was available

**What did you expect to happen?**
Video to start playing

**What actually happened?**
Video never plays

This issue appeared as of v2.2.3 and I've narrowed it down to the fix for https://github.com/google/shaka-player/issues/1061 (commit https://github.com/google/shaka-player/commit/449cd3ebef20208b9f807e624d18c4b9965a324f).  It looks like if you update that to  only return from `onPollGapJump_()` while seeking and not from a segment getting appended it fixes the issue:

```diff
diff --git a/lib/media/playhead.js b/lib/media/playhead.js
index 3c4e1c8..9200877 100644
--- a/lib/media/playhead.js
+++ b/lib/media/playhead.js
@@ -347,7 +347,7 @@ shaka.media.Playhead.prototype.onPollGapJump_ = function() {
   // Don't gap jump before the video is ready to play.
   if (this.video_.readyState == 0) return;
   // Don't gap jump while seeking, to prevent a race condition.
-  if (this.video_.seeking) return;
+  if (this.video_.seeking && !this.hadSegmentAppended_) return;
   // Don't gap jump while paused, so that you don't constantly jump ahead while
   // paused on a livestream.
   if (this.video_.paused) return;
```

**Comment by TheModMaker** [comment]
We just reverted the fix for #1061.  Can you test again to see if that fixes it?  You can use the nightly, but be sure to clear the cache and check the version starts with `v2.2.0` followed by a number that is at least 133:

https://nightly-dot-shaka-player-demo.appspot.com/demo/

**Comment by crazytoad** [comment]
I can confirm the issue is fixed in `v2.2.0-133-g59b48f0` - thank you!

**Comment by joeyparrish** [comment]
Fix cherry-picked for v2.2.7.
